src/main/kotlin/no/nav/syfo/dialogmote/database/MotedeltakerQuery.kt

```diff
@@ -57,13 +58,20 @@ const val queryGetMotedeltakerArbeidstakerForMote =
         WHERE mote_id = ?
     """
 
-fun DatabaseInterface.getMoteDeltakerArbeidstaker(moteId: Int): List<PMotedeltakerArbeidstaker> {
-    return this.connection.use { connection ->
+fun DatabaseInterface.getMoteDeltakerArbeidstaker(moteId: Int): PMotedeltakerArbeidstaker {
+    val pMotedeltakerArbeidstakerList = this.connection.use { connection ->
         connection.prepareStatement(queryGetMotedeltakerArbeidstakerForMote).use {
             it.setInt(1, moteId)
             it.executeQuery().toList { toPMotedeltakerArbeidstaker() }
         }
     }
+    if (pMotedeltakerArbeidstakerList.isEmpty()) {
+        throw RuntimeException("No motedeltakerArbeidstaker found for mote with id $moteId")
+    }
+    if (pMotedeltakerArbeidstakerList.size > 1) {
+        throw RuntimeException("More than one motedeltakerArbeidstaker found for mote with id $moteId")
+    }
+    return pMotedeltakerArbeidstakerList.first()
 }
 
```

Comment:
Ser det er et par andre metoder her (getMoteDeltakerArbeidsgiver og getMotedeltakerArbeidstakerById) som også returnerer liste, men bare .first() brukes - kan vurdere å endre de tilsvarende. 
getMoteDeltakerArbeidstaker(personIdentNumber: PersonIdentNumber) er ubrukt og kan slettes.

Reply:
Ja, her er det sikkert bra med litt mer opprydning. Generelt så har vi en modell om tillater flere forekomster databasemessig, men egentlig forventer vi at det bare skal være en forekomst. Man må bruke en liste når man leser ut fra databasen, men jeg synes det da er bedre å validere i databaselaget at det er en unik forekomst - i stedet for å at vi sender lister med et enkelt element videre.

Kan gjøre litt mer tilsvarende refaktorering i en annen PR.